src/platform/intel/ace/platform.c

```diff
@@ -98,6 +115,9 @@ int platform_init(struct sof *sof)
 	if (ret < 0)
 		return ret;
 
+	/* register power states entry / exit notifiers */
+	pm_notifier_register(&pm_state_notifier);
```

Comment:
Is long term switch over to AMS here ?

Reply:
No, it will stay as it is (if nothing changes).